Core/Core/View/Base/ScrollSlidingTabBar/ScrollSlidingTabBar.swift

```diff
@@ -0,0 +1,241 @@
+                    isSelected(index: obj.offset) ? style.activeAccentColor : style.inactiveAccentColor
+                )
+                .readFrame(in: .named(containerSpace)) {
+                    buttonFrames[obj.offset] = $0
+                }
+                .id(obj.offset)
+            }
+        }
+    }
+    
+    private func indicatorContainer() -> some View {
+        Rectangle()
+            .fill(Color.clear)
+            .frame(width: tabWidth(), height: style.indicatorHeight)
+            .overlay(indicator(), alignment: .center)
+            .offset(x: selectionBarXOffset(), y: 0)
+            .animation(.default, value: selection)
+    }
+    
+    private func indicator() -> some View {
+        Rectangle()
+            .fill(style.activeAccentColor)
+            .frame(width: indicatorWidth(selection: selection), height: style.indicatorHeight)
+    }
+}
+
+extension ScrollSlidingTabBar {
+    private func sanitizedSelection() -> Int {
+        return max(0, min(tabs.count - 1, selection))
+    }
+    
+    private func isSelected(index: Int) -> Bool {
+        return sanitizedSelection() == index
+    }
+    
+    private func selectionBarXOffset() -> CGFloat {
+        return buttonFrames[sanitizedSelection()]?.minX ?? .zero
+    }
+    
+    private func indicatorWidth(selection: Int) -> CGFloat {
+        return max(tabWidth() - style.buttonHInset * 2, .zero)
+    }
+    
+    private func tabWidth() -> CGFloat {
+        return buttonFrames[sanitizedSelection()]?.width ?? .zero
+    }
+}
+
+extension ScrollSlidingTabBar {
+    public struct Style {
+        public let font: Font
+        public let selectedFont: Font
+        
+        public let activeAccentColor: Color
+        public let inactiveAccentColor: Color
+        
+        public let indicatorHeight: CGFloat
+        
+        public let borderColor: Color
+        public let borderHeight: CGFloat
+        
+        public let buttonHInset: CGFloat
+        public let buttonVInset: CGFloat
+        
+        public init(
+            font: Font,
+            selectedFont: Font,
+            activeAccentColor: Color,
+            inactiveAccentColor: Color,
+            indicatorHeight: CGFloat,
+            borderColor: Color,
+            borderHeight: CGFloat,
+            buttonHInset: CGFloat,
+            buttonVInset: CGFloat
+        ) {
+            self.font = font
+            self.selectedFont = selectedFont
+            self.activeAccentColor = activeAccentColor
+            self.inactiveAccentColor = inactiveAccentColor
+            self.indicatorHeight = indicatorHeight
+            self.borderColor = borderColor
+            self.borderHeight = borderHeight
+            self.buttonHInset = buttonHInset
+            self.buttonVInset = buttonVInset
+        }
+        
+        public static let `default` = Style(
+            font: .body,
+            selectedFont: .body.bold(),
+            activeAccentColor: .blue,
```

Comment:
@eyatsenkoperpetio  I think we need to use Theme.Colors.accentColor instead of just .blue